apps/builder/app/builder/features/sidebar-left/help-popover.tsx

```diff
@@ -0,0 +1,110 @@
+import {
+  Button,
+  Flex,
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+  styled,
+  theme,
+} from "@webstudio-is/design-system";
+import {
+  DiscordIcon,
+  GithubIcon,
+  TwitterIcon,
+  WebstudioLogoFlatIcon,
+  YoutubeIcon,
+} from "@webstudio-is/icons";
+import { type ComponentProps } from "react";
+
+const StyledLink = styled("a", {
+  textDecoration: "none",
+  color: "white",
+  display: "inline-flex",
+  cursor: "default",
+});
+
+const PopoverItemButton = ({
+  href,
+  ...buttonProps
+}: ComponentProps<typeof Button> & { href?: string }) => (
+  <StyledLink href={href} target="_blank" referrerPolicy="no-referrer">
+    <Button
+      css={{
+        width: "100%",
+      }}
+      {...buttonProps}
+    />
+  </StyledLink>
+);
+
+export const HelpPopover = ({
+  children,
+  side = "right",
+  ...popoverProps
+}: {
+  children: React.ReactNode;
+  side?: "bottom" | "left" | "right" | "top";
+} & ComponentProps<typeof Popover>) => {
+  return (
+    <Popover {...popoverProps}>
+      {children}
```

Comment:
I can see that trigger could be wrapping children here automatically, then you would simplify even the consumer side

Reply:
The problem why I didn't used the trigger here is in the `sidebar-left` component. The tooltip consider the popover as the tooltip trigger.